crypto/openssl/tpm20_Utils.c

```diff
@@ -594,11 +606,17 @@ static int32_t fdoTPMEsys_auth_session_init(ESYS_CONTEXT *esys_context,
 	int ret = -1;
 	TPMT_SYM_DEF symmetric = {0};
 	symmetric.algorithm = TPM2_ALG_NULL;
-
+#if defined(ECDSA256_DA)
 	TSS2_RC rval = Esys_StartAuthSession(
 	    esys_context, ESYS_TR_NONE, ESYS_TR_NONE, ESYS_TR_NONE,
 	    ESYS_TR_NONE, ESYS_TR_NONE, NULL, TPM2_SE_HMAC, &symmetric,
 	    TPM2_ALG_SHA256, session_handle);
+#else
```

Comment:
Can we define TPM2_ALG_SHA_TYPE as TPM2_ALG_SHA256 or TPM2_ALG_SHA384 based on "defined(ECDSA256_DA)" at one place( like how u defined PLATFORM_HMAC_SIZE) ? I hope then we can remove the multiple #ifs